### DISCUSSION TITLE
FirebaseAuth + NyloAuth

### DISCUSSION THREAD

**Discussion by AvazbekNadyrbek** [nylo-core/nylo#202] [Q&A] [answered]
Dear Nylo developers and community,

I'm very impressed with Nylo's microframework for Flutter. I'm currently learning about its authentication features. I'm curious if it's possible to integrate Firebase Authentication with Nylo, and if so, how? I've looked online but haven't found any examples. Could you provide guidance or point me to resources that explain this integration? Thanks in advance!

**Comment by agordn52** [comment, marked as answer]
Hi @AvazbekNadyrbek, 

Welcome and thanks for the kind words. 

You'll need to initialise Firebase as you would in any other Flutter project. However, I'd suggest initialising it in a Provider. 

You can learn more here https://nylo.dev/docs/6.x/providers

In the future we might add Firebase auth to the scaffold ui package.
